changelogs/fragments/58812-support_absolute_paths_additionally.yml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
In case you want your PR to be a bugfix, you should change this to `bugfixes:`.

(In that case, I can create a backport PR for you so it has a chance of being backported to stable-2.9 and end up in 2.9.x.)

Reply:
fixed